modules/learn/pages/clusters-and-availability/xdcr-enable-crossclusterversioning.adoc

```diff
@@ -0,0 +1,65 @@
+= XDCR enableCrossClusterVersioning
+:description: pass:q[Enabling the bucket property `enableCrossClusterVersioning` allows XDCR to add metadata to each replicated document.]
+
+[abstract]
+{description}
+
+
+Enabling the bucket property `enableCrossClusterVersioning` for all buckets in the replication topology is a pre-requisite for some XDCR features. The bucket property `enableCrossClusterVersioning` cannot be disabled once it is set to `true`. Therefore, you must understand all the pre-requisites and the full effects of setting `enableCrossClusterVersioning` to `true` before enabling it.
+
+When you set the bucket property `enableCrossClusterVersioning` (ECCV) to `true`, for each document processed by XDCR, XDCR stores additional metadata for the document in the extended attributes. This metadata is also called Hybrid Logical Vector (HLV), which is a set of Hybrid Logical Clock (HLC) information.
+
+[#dependent-xdcr-features]
+== Dependent XDCR features
```

Comment:
This section should be moved to the end of the page.  This way, the above mention of HLV can flow into the section called "Hybrid Logical Vector (HLV) data maintained in the extended attributes".

Also, instead of calling it "Dependent XDCR features", let's call it "Features that require enabling Cross Cluster Versioning", and we can just list the Sync Gateway/XDCR feature with a link to the Sync Gateway/XDCR page that you created.